### PR TITLE
Update predict-fl.md

### PR DESCRIPTION
**Pull request by hfleitas**
added model asset and dataset to test adhoc script outside of help cluster, which doesnt have python pluging enabled nor ML_Models table.

### CODE REVIEW

**Comment by shsagir** [comment]
Learn Build status updates of commit _[8142a60](https://github.com/hfleitas/dataexplorer-docs/commits/8142a60bc2dfb3b8971d01028fe430272f03368a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/kusto/functions-library/predict-fl.md](https://github.com/hfleitas/dataexplorer-docs/blob/patch-2/data-explorer/kusto/functions-library/predict-fl.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202210082152304447-1442/BuildReport?accessString=af37fa9a144d84451a9007e598038109b0df833eec19148dbd7dd98b29452b30).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by Court72** [comment]
@shsagir 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by YaelSchuster** [comment]
Approved by @adieldar 

#sign-off

File: data-explorer/kusto/functions-library/predict-fl.md [hunk 1]
```diff
@@ -139,3 +139,18 @@ FALSE	    TRUE	        112
 TRUE	    FALSE	        15
 FALSE	    FALSE	        9284
 ```
+
+### Model asset
+
+Get sample dataset and pre-trained model in your cluster with python plugin enabled.
```
Comment (Court72) [inline data-explorer/kusto/functions-library/predict-fl.md:145]:
```suggestion
Get sample dataset and pre-trained model in your cluster with Python plugin enabled.
```

**Comment by shsagir** [comment]
Learn Build status updates of commit _[c3d65cb](https://github.com/hfleitas/dataexplorer-docs/commits/c3d65cbeeb9c1ea05338cac6407cdef6dc7debd8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/kusto/functions-library/predict-fl.md](https://github.com/hfleitas/dataexplorer-docs/blob/patch-2/data-explorer/kusto/functions-library/predict-fl.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202211071528568371-1442/BuildReport?accessString=169abf58cd06c1ee0b54a08759fe0aadfb1028d67c9b8fbf578367deac299580).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Pull request merged**